packages/katello/katello/hostname-change.rb

```diff
@@ -143,8 +143,7 @@ def next_steps_message
   You will have to install the new bootstrap rpm and reregister all clients and #{@plural_proxy} with subscription-manager
   (update organization and environment arguments appropriately):
 
-  yum remove -y katello-ca-consumer*
-  rpm -Uvh http://#{@new_hostname}/pub/katello-ca-consumer-latest.noarch.rpm
+  set -o pipefail && curl -sS "http://#{@new_hostname}/unattended/public/foreman-ca-refresh" | bash
   subscription-manager register --org="Default_Organization" --environment="Library" --force
```

Comment:
Is this the default way to register? I think, using a activation key is more common. 

Or should is use the HostRegistration template for the whole process?

Reply:
Host registration is actually a good idea, but we can't put here the command that initiates it, since each invocation generates a new token.
Maybe we should change this part to something that will say "Go to Foreman UI and generate a registration command" (with proper links of course)?

Reply:
As I stated above, changing the hostname should not require clients to be re-registered, only re-configured.